magma/lib/magma/attribute.rb

```diff
@@ -96,8 +96,7 @@ def json_template
         attribute_name: attribute_name,
         model_name: self.is_a?(Magma::Link) ? link_model.model_name : nil,
         link_model_name: self.is_a?(Magma::Link) ? link_model.model_name : nil,
-        type: database_type.respond_to?(:name) ? database_type.name : database_type,
-        attribute_class: attribute_class_name,
```

Comment:
I checked our clients, and this seems safe.  All our workflow rely on `attribute_type` and not either of these two.